frontend/Dockerfile.dev

```diff
@@ -2,6 +2,8 @@ FROM node:14-alpine
 
 ENV NODE_ENV=development
 
+RUN apk --no-cache add git
+
```

Comment:
What's that for?

Reply:
Installing `1-click-bom` https://github.com/kitspace/kitspace-v2/blob/e4e9ab98ed31d8bc61246ce5b69570dec681323c/frontend/package.json#L13

Reply:
Installing `1-click-bom` https://github.com/kitspace/kitspace-v2/blob/e4e9ab98ed31d8bc61246ce5b69570dec681323c/frontend/package.json#L13

Reply:
Can you add some comments please, so know we can remove it when we no longer have git dependencies.

Reply:
Whenever I install a new dependency I add the why to the commit, see https://github.com/kitspace/kitspace-v2/commit/bc5474254f25cf772d31f1e7a9ffa4939c298b7d

Reply:
Can you add comment to the Dockerfile please. We may move/rename the Dockerfile and the git comment will be hard to track down. We need a little reminder to remove the git install from the Dockerfile if we don't have git dependencies.

Reply:
Resolved via d168558.